client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import { map, orderBy } from 'lodash';
+import React, { PureComponent, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import QueryPostRevisions from 'components/data/query-post-revisions';
+import EditorRevisionsListHeader from './header';
+import {
+	getPostRevision,
+	getPostRevisions,
+	normalizeForDisplay,
+	normalizeForEditing,
+} from 'state/posts/revisions/selectors';
+import EditorRevisionsListItem from './item';
+
+class EditorRevisionsList extends PureComponent {
+	constructor() {
+		super();
+		this.loadRevision = this.loadRevision.bind( this );
+	}
+
+	loadRevision() {
+		this.props.loadRevision( this.props.selectedRevision );
+	}
+
+	componentWillMount() {
+		if (
+			this.props.selectedRevisionId === null &&
+			this.props.revisions.length > 0
+		) {
+			this.props.toggleRevision( this.props.revisions[ 0 ].id );
+		}
+	}
+
+	render() {
+		return (
+			<div>
+				<QueryPostRevisions siteId={ this.props.siteId } postId={ this.props.postId } />
+				<EditorRevisionsListHeader loadRevision={ this.loadRevision } />
+				<ul className="editor-revisions-list__list">
+					{ map( this.props.revisions, revision => (
+						<li
+							className={ classNames(
```

Comment:
Doing the classnames construct outside the return call often helps with readability.